Library/Homebrew/test/x11_requirement_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "requirements/x11_requirement"
+
+describe X11Requirement do
+  subject { described_class.new }
+
+  describe "#eql?" do
```

Comment:
Well, I don't know what was originally intended to be checked, but that's what I'm guessing should be checked.